app/policies/result_policy.rb

```diff
@@ -0,0 +1,9 @@
+class ResultPolicy < ApplicationPolicy
+  def new?
```

Comment:
If you take a look at application policy, you'll see that new is defined as `create?`. If you define `create?` you will get `new?` for free.